src/dds/rs-dds-option.cpp

```diff
@@ -126,6 +126,9 @@ void rs_dds_option::set( float value )
                                          << "use rs2_set_option_value to set " << get_string( _rs_type ) << " value" );
     }
 
+    if( is_read_only() )
```

Comment:
It's a good question. Like I said in the description:
Librealsense on USB seems to depend on the FW to complain.
I don't see a reason, at least for DDS, to go to the server when the server has indicated the option is read-only.
I only did this because I was surprised when my unit-test failed. Seems weird to have to implement server-side code to complain when it clearly said an option is read-only.
But I'll accept your opinion here.

Reply:
I accept that sending over the net when it is marked read only is an overhead.
Leave the check.